api/src/bigquerytest/java/org/pmiops/workbench/api/CohortReviewControllerBQTest.java

```diff
@@ -209,57 +207,47 @@ public void setUp() {
     stubMockFirecloudGetWorkspaceAcl();
 
     Gson gson = new Gson();
-    cohortWithoutEHRData = new DbCohort();
+    DbCohort cohortWithoutEHRData = new DbCohort();
     cohortWithoutEHRData.setWorkspaceId(workspace.getWorkspaceId());
     cohortWithoutEHRData.setCriteria(gson.toJson(SearchRequests.males()));
     cohortWithoutEHRData = cohortDao.save(cohortWithoutEHRData);
 
-    cohortWithEHRData = new DbCohort();
+    DbCohort cohortWithEHRData = new DbCohort();
     cohortWithEHRData.setWorkspaceId(workspace.getWorkspaceId());
     cohortWithEHRData.setCriteria(gson.toJson(SearchRequests.malesWithEHRData()));
     cohortWithEHRData = cohortDao.save(cohortWithEHRData);
 
-    reviewWithoutEHRData =
-        new DbCohortReview()
-            .cdrVersionId(cdrVersion.getCdrVersionId())
-            .matchedParticipantCount(212)
-            .creationTime(new Timestamp(new Date().getTime()))
-            .lastModifiedTime(new Timestamp(new Date().getTime()))
-            .cohortId(cohortWithoutEHRData.getCohortId());
-    reviewWithoutEHRData = cohortReviewDao.save(reviewWithoutEHRData);
-
-    DbParticipantCohortStatusKey key =
-        new DbParticipantCohortStatusKey()
-            .participantId(PARTICIPANT_ID)
-            .cohortReviewId(reviewWithoutEHRData.getCohortReviewId());
-    participantCohortStatusDao.save(new DbParticipantCohortStatus().participantKey(key));
-
-    DbParticipantCohortStatusKey key2 =
-        new DbParticipantCohortStatusKey()
-            .participantId(PARTICIPANT_ID2)
-            .cohortReviewId(reviewWithoutEHRData.getCohortReviewId());
-    participantCohortStatusDao.save(new DbParticipantCohortStatus().participantKey(key2));
-
-    reviewWithEHRData =
-        new DbCohortReview()
-            .cdrVersionId(cdrVersion.getCdrVersionId())
-            .matchedParticipantCount(212)
-            .creationTime(new Timestamp(new Date().getTime()))
-            .lastModifiedTime(new Timestamp(new Date().getTime()))
-            .cohortId(cohortWithEHRData.getCohortId());
-    reviewWithEHRData = cohortReviewDao.save(reviewWithEHRData);
-
-    DbParticipantCohortStatusKey key3 =
-        new DbParticipantCohortStatusKey()
-            .participantId(PARTICIPANT_ID)
-            .cohortReviewId(reviewWithEHRData.getCohortReviewId());
-    participantCohortStatusDao.save(new DbParticipantCohortStatus().participantKey(key3));
-
-    DbParticipantCohortStatusKey key4 =
-        new DbParticipantCohortStatusKey()
-            .participantId(PARTICIPANT_ID2)
-            .cohortReviewId(reviewWithEHRData.getCohortReviewId());
-    participantCohortStatusDao.save(new DbParticipantCohortStatus().participantKey(key4));
+    reviewWithoutEHRData = createCohortReview(cohortWithoutEHRData);
+
+    participantCohortStatusDao.save(
+        new DbParticipantCohortStatus()
```

Comment:
this could be moved to a method (maybe pass Participant Id) and called again

Reply:
done.